ddmrp/models/stock_warehouse_orderpoint.py

```diff
@@ -371,20 +406,6 @@ def open_moves(self):
         records = self.env['stock.move'].search(domain)
         return self._stock_move_tree_view(records)
 
-    @api.model
-    def subtract_procurements(self, orderpoint):
-        qty = super(StockWarehouseOrderpoint, self).subtract_procurements(
-            orderpoint)
-        for procurement in orderpoint.procurement_ids:
-            if procurement.state not in ('draft', 'cancel') and \
-                    procurement.add_to_net_flow_equation:
-                qty += procurement.product_uom._compute_quantity(
-                    procurement.product_qty, procurement.product_id.uom_id)
-        if qty >= 0.0:
-            return qty
-        else:
-            return 0.0
-
     @api.model
```

Comment:
This should be `@api.multi`
Same at lines 424, 453, 460, 372.
I didn't check other classes yet.